docs/installation/sources/sources_windows.rst

```diff
@@ -379,12 +382,232 @@ Run an application
 ^^^^^^^^^^^^^^^^^^
 
 When running an instance of an application using *eProsima Fast DDS*, it must be linked with the library where the
-packages have been installed. This can be done by opening the *Edit system environment variables* control panel and
-adding to the ``PATH`` the *Fast DDS* and *Fast CDR* installation directories:
+packages have been installed.
+This can be done by opening the *Edit system environment variables* control panel and adding to the ``PATH`` the
+*Fast DDS* and *Fast CDR* installation directories:
 
 *   *Fast DDS*: C:\\Program Files\\fastrtps
 *   *Fast CDR*: C:\\Program Files\\fastcdr
 
+.. _fastdds_python_sw:
+
+Fast DDS Python bindings installation
+"""""""""""""""""""""""""""""""""""""
+
+This section provides the instructions for installing *Fast DDS Python bindings* in a Windows environment from sources.
+*Fast DDS Python bindings* is an extension of *Fast DDS* which provides access to the Fast DDS API through Python.
+Therefore, its installation is an extension of the installation of :ref:`Fast DDS <fastdds_lib_sw>`.
+
+*Fast DDS Python bindings* source code consists on several `.i` files which will be processed by SWIG_.
+Then C++ files (for connecting C++ and Python) and Python files (Python module for Fast DDS) will be generated.
+
+First of all, the :ref:`requirements_python_sw` and :ref:`dependencies_python_sw` detailed below need to be met.
+Afterwards, the user can choose whether to follow either the :ref:`colcon <colcon_installation_python_windows>` or the
+:ref:`CMake <cmake_installation_python_windows>` installation instructions.
+
+.. _requirements_python_sw:
+
+Requirements
+------------
+
+The installation of *Fast DDS Python bindings* in a Windows environment from sources requires the following tools to be
+installed in the system:
+
+- :ref:`Fast DDS requirements <requirements_sw>`
+- :ref:`swig_python_sw`
+- :ref:`libpython_dev_sw`
```

Comment:
I think this is not required on Windows. Installing `python` through `chocolatey` should be enough.